<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputAlignmentAreaFormClass</class>
 <widget class="QDialog" name="InputAlignmentAreaFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InputAlignmentAreaForm</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>エリア名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditAreaName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>ライブラリごとのアラインメント</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frameLibrary">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>451</width>
     <height>361</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonOK">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>560</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyAlignment.qrc">
     <normaloff>:/Qtres/OK.png</normaloff>:/Qtres/OK.png</iconset>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>560</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyAlignment.qrc">
     <normaloff>:/Qtres/Cancel.png</normaloff>:/Qtres/Cancel.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>560</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyAlignment.qrc">
     <normaloff>:/Qtres/Delete.png</normaloff>:/Qtres/Delete.png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Priority</string>
   </property>
   <widget class="QRadioButton" name="radioButtonPriorityHigh">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>251</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>High( Exp. BGA, special part)</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonPriorityMiddle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Middle (General)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonPriorityLow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Low (Only parts fitting)</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PropertyAlignment.qrc"/>
 </resources>
 <connections/>
</ui>
